Mathlib/Order/Category/BoolAlg.lean

```diff
@@ -176,9 +176,6 @@ def dual : BoolAlg ⥤ BoolAlg where
   obj X := of Xᵒᵈ
   map f := ofHom f.hom.dual
 
-#adaptation_note /-- nightly-2025-02-12
-This apparently became slower. (Possibly due to changes in `simp +arith`?) -/
-set_option maxHeartbeats 400000 in
```

Comment:
I hope this change (and its analogues) are safe, in the sense that the declarations are now not close to the limit.

Reply:
`#count_heartbeats` says 52622.